application/command.py

```diff
@@ -0,0 +1,24 @@
+"""このスクリプトは、コマンドラインから特定のコマンドを実行するためのものです。.
+
+コマンドライン引数に応じて、異なる機能を実行します。
+
+使い方:
+    python [スクリプト名] [コマンド]
+
+利用可能なコマンド:
+    load_json: JSONファイルを読み込み、指定された処理を実行します。
+
+注意:
+    コマンドライン引数が適切でない場合、エラーメッセージを表示します。
+"""
+import asyncio
+import sys
+
+from command import load_json
+
+if __name__ == "__main__":
+    args = sys.argv
+    if len(args) < 2:
+        print("実行するコマンドを指定していください", file=sys.stderr)
```

Comment:
細かいですがtypoしてますので、修正お願いします🤲

```suggestion
        print("実行するコマンドを指定してください", file=sys.stderr)
```

Reply:
修正しました。
[da9b64a](https://github.com/furutahidehiko/mobile_reader_novel_back/pull/11/commits/da9b64a8e4083218f658938f88ecfd70b9c14536)